example.yaml

```diff
@@ -36,8 +36,14 @@ tls_authority_certificate: ""
 
 # Use a static host for forwarding
 #forward_address: "http://veneur.example.com"
+# Do not add a prefix when setting the forward address for gRPC
+#forward_address: "veneur.example.com"
```

Comment:
Sure, this seems reasonable to me.  I made this change in 3d3e669, which adds a new `grpc_forward_address` parameter which is exclusive with `forward_address` (which retains its functionality as setting the HTTP forward address).  I'm happy to change that parameter to `http_forward_address` if you'd like (which would definitely be more clear), I just didn't want to do it initially since its a particularly breaking change to existing configurations.

I decided to retain the set of struct fields that I previously added to `Server`, specifically the `forwardUseGRPC` bool, as well as using `ForwardAddr` for both addresses.  My thought was that this will be a little easier in the long run if HTTP forwarding is ever removed, and that using a simple boolean switch to check which forwarding type is being used is very simple.  I'm happy to change to use two separate addresses as well though, there doesn't seem like a big difference to me!